### PR TITLE
Update line length value

### PR DESCRIPTION
**Pull request by big-r81**

### CODE REVIEW

File: src/contributing.rst [hunk 1]
```diff
@@ -192,7 +192,7 @@ The guidelines are in descending priority.
 
 #. Line length
 
-   * The maximum line length is ``80`` characters.
+   * The maximum line length is ``90`` characters.
```
Comment (nickva) [inline src/contributing.rst:195]: I wonder if we technically limit it at that in erlfmt or just have the 100 character limit.

Comment (big-r81) [reply src/contributing.rst:195]: This limit is only for the documentation…

Comment (big-r81) [reply src/contributing.rst:195]: See https://github.com/apache/couchdb-documentation/blob/f57d9b2ca93342da94365ae602e57635106efdf9/ext/linter.py#L206

Comment (nickva) [reply src/contributing.rst:195]: Ah, perfect. I confused that with the main repo

**Review by nickva — APPROVED** [review]

**Pull request merged**